src/olympia/addons/models.py

```diff
@@ -16,6 +17,7 @@
 from django.db.models import Max, Q, signals as dbsignals
 from django.utils.translation import trans_real, ugettext_lazy as _
 
+import waffle
```

Comment:
Nitpick, but should this be at the top? Seems we do imports in alphabetical order.